spec/features/finance/payment_breakdown_spec.rb

```diff
@@ -91,6 +103,13 @@ def and_multiple_declarations_are_submitted
     multiple_retained_declarations_are_submitted_jan_statement
   end
 
+  def and_voided_payable_declarations_are_submitted
+    participant_profiles = create_list(:ect_participant_profile, 2, school_cohort: school_cohort, cohort: cohort, sparsity_uplift: true)
+    participant_profiles.map { |participant| ParticipantProfileState.create!(participant_profile: participant) }
+    participant_profiles.map { |participant| ECFParticipantEligibility.create!(participant_profile_id: participant.id).eligible_status! }
+    participant_profiles.map { |participant| create_voided_declarations_nov(participant) }
+  end
```

Comment:
Should probably use `EarlyCareerTeachers::Create` for a realistic test setup. 
Probably would not be more concise as the you'd have to setup a school on the school pupil premium and so on... But it would be worth investing the time to get some school factories that allow for this.

Reply:
I'm jumping to on the bandwagon of what was already here. I don't think it's worth the effort tidying. I think in the long term all these tests should be ripped out. The view should be getting some kind of view object that we can stub and test. We should not be doing huge and complex data setups just to test a view.

Reply:
Well that's sort of our "end to end" test. So a realistic test setup provides much great assurance the test is valuable. Otherwise you just have a test that you think might work if your setup mirrors the production code. But we won't have this screen for long, might point was just to re-use production code for test setup which is far more reliable than having hand crafted test setup which intrinsic correctness will drift as the production code evolves.